scripts/create-kubestellar-demo-env.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+# The formal instructions accompaning this script can be found at:
+# https://docs.kubestellar.io/release-0.23.1/direct/get-started/
+
+set -e
+
+echo -e "Checking that pre-req softwares are installed..."
+curl -s -o check_pre_req.sh https://raw.githubusercontent.com/kubestellar/kubestellar/v0.23.1/hack/check_pre_req.sh
+chmod +x check_pre_req.sh
+./check_pre_req.sh -V kflex ocm helm kubectl docker kind
+
+output=$(./check_pre_req.sh -V kflex ocm helm kubectl docker kind)
+
+if echo "$output" | grep -q "X"; then
+    echo "Please install all the necessary dependencies before contiuning"
+    exit 1
+else
+    echo "Pre-req are properly installed. Proceeding to environment clean up"
+fi
+
+echo -e "\nStarting environment clean up..."
+echo -e "Starting cluster clean up..."
+
+cluster_clean_up() {
+    error_message=$(eval "$1" 2>&1)
+    if [ $? -ne 0 ]; then
+        echo "clean up failed. Error:"
+        echo "$error_message"
+    fi
+}
+
+cluster_clean_up "kind delete cluster --name kubeflex"
+cluster_clean_up "kind delete cluster --name cluster1"
+cluster_clean_up "kind delete cluster --name cluster2"
+echo -e "Cluster space clean up has been completed"
+
+echo -e "\nStarting context clean up..."
+
+context_clean_up() {
+    output=$(kubectx)
```

Comment:
What is `kubectx` ?

Reply:
kubectx is a tool to switch between contexts (clusters) on kubectl faster.

Reply:
```shell
mspreitz@mjs13 kubestellar % which kubectx
kubectx not found
```

`kubectx` is not among the listed prerequites for using KubeStellar.

Reply:
i changed it to use a kubectl command instead

Reply:
Did you push your change? I still see `kubectx` being used.